snippets/juxt.md

```diff
@@ -0,0 +1,28 @@
+---
+title: juxt
+tags: array,function,advanced
+---
+
+Takes several functions as argument and returns a fn that is the juxtaposition of those fns.
+
+juxt(fn1, fn2, fn3, ...) use `Array.prototype.map()` to return a `func` that can take a variable number of `args`.
```

Comment:
```suggestion
Use `Array.prototype.map()` to return a `fn` that can take a variable number of `args`.
```

Also, I think you should rename `x` in the code to `args` to be consistent with the description.